ApollosProject/apollos-prototype
# Eliminates inconsistencies for Avatar Photo upload

## DESCRIPTION

### What does this PR do, or why is it needed?

In the QA push, it was noticed that when updating a profile picture, it would not show up in the `settings` view and was inconsistently showing up in the `Connect` view. 

### What design trade-offs/decisions were made?

This PR does reduce the `quality` of the photo that is loaded in through the `react-native-image-picker`.

### How do I test this PR?
Navigate to the `Connect` view. Ensure that you're signed in. Then click `change photo` and proceed with selecting a new photo.

### What automated tests did you write?

N/A

## TODO:

- [x] Closes [#385]

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
